src/main/io/gps.c

```diff
@@ -1115,6 +1624,7 @@ typedef struct gpsDataNmea_s {
     uint16_t ground_course;
     uint32_t time;
     uint32_t date;
+    uint16_t secondsInTenths;
```

Comment:
deciSeconds ?

Reply:
I can avoid using this extra field, and will use milliseconds.